test-common/utils.c

```diff
@@ -2932,7 +2932,7 @@ eos_test_client_prepare_volume (EosTestClient *client,
                                                                                "eos-updater-prepare-volume",
                                                                                "eos-updater-prepare-volume",
                                                                                NULL);
-  g_autofree gchar *libeos_updater_util_path = g_test_build_filename (G_TEST_DIST,
+  g_autofree gchar *libeos_updater_util_path = g_test_build_filename (G_TEST_BUILT,
```

Comment:
But that folder is distributed not built? I don't understand this change.

Reply:
The path which is produced here is set as the `GI_TYPELIB_PATH` in the environment of `eos-updater-prepare-volume`. The `.typelib` files are built at build time, and are not distributed — so they will be present only in the `G_TEST_BUILT` path for `libeos-updater-util`.